{
	"id": "SUSE-SU-2024:4396-1",
	"modified": "2024-12-20T12:01:46Z",
	"published": "2024-12-20T12:01:46Z",
	"related": [
		"CVE-2024-27306"
	],
	"summary": "Security update for python-aiohttp",
	"details": "This update for python-aiohttp fixes the following issues:\n\n- CVE-2024-27306: filenames and paths not escaped when generating index pages for static file handling. (bsc#1223098)\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 15 SP2",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/suse/python-aiohttp&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2015%20SP2"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-aiohttp-doc": "3.6.0-150100.3.24.1",
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 15 SP3",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/suse/python-aiohttp&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2015%20SP3"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 15 SP4",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/suse/python-aiohttp&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2015%20SP4"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 15 SP5",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/suse/python-aiohttp&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2015%20SP5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 15 SP6",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/suse/python-aiohttp&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2015%20SP6"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.5",
				"name": "python-aiohttp",
				"purl": "pkg:rpm/opensuse/python-aiohttp&distro=openSUSE%20Leap%2015.5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.0-150100.3.24.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-aiohttp-doc": "3.6.0-150100.3.24.1",
						"python3-aiohttp": "3.6.0-150100.3.24.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20244396-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1223098"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-27306"
		}
	]
}
